docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "db0b5179",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sentence 2907: score=0.000001\n",
+      "\u001b[31mLittle\u001b[0m change from today's weather expected.\n",
+      "Given label: PER, suggested label: O\n",
+      "\n",
+      "Sentence 19392: score=0.000001\n",
+      "\u001b[31mLet\u001b[0m's march together,\" Scalfaro, a northerner himself, said.\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "Sentence 9962: score=0.000001\n",
+      "3. Nastja Rysich (\u001b[31mgermany\u001b[0m) 3.75\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "Sentence 8904: score=0.000002\n",
+      "The Spla has fought Khartoum's government forces in the south since 1983 for greater autonomy or independence of the mainly Christian and animist region from the Moslem, Arabised \u001b[31mnorth\u001b[0m.\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "Sentence 19303: score=0.000002\n",
+      "\u001b[31mAccess\u001b[0m energy futures prices add to daytime gains.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "Sentence 12918: score=0.000002\n",
+      "\u001b[31mMayor\u001b[0m Antonio Gonzalez Garcia, of the opposition Revolutionary Workers' Party, said in Wednesday's letter that army troops recently raided several local farms, stole cattle and raped women.\n",
+      "Given label: PER, suggested label: O\n",
+      "\n",
+      "Sentence 9256: score=0.000002\n",
+      "\u001b[31mSpring\u001b[0m Chg Hrw 12pct Chg White Chg\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "Sentence 11855: score=0.000002\n",
+      "\" We have seen the photos but for the moment the palace has no comment,\" a spokeswoman for \u001b[31mPrince\u001b[0m Rainier told Reuters.\n",
+      "Given label: PER, suggested label: O\n",
+      "\n",
+      "Sentence 18392: score=0.000002\n",
+      "Danila 28.5 16\u001b[31m/\u001b[0m12 Caribs/ up W224 Mobil.\n",
+      "Given label: O, suggested label: LOC\n",
+      "\n",
+      "Sentence 20426: score=0.000002\n",
+      "Albanian coach Astrit Hafizi said on Saturday it was important that his players brush aside the country's short ban by Fifa in order to concentrate on next \u001b[31mSaturday'sWorld\u001b[0m Cup group nine qualifier against Northern Ireland.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "scores, token_scores = get_label_quality_scores(labels, pred_probs, return_token_info=True) \n",
+    "show_sentence_issues(scores, token_scores, pred_probs, given_words, sentences, given_labels, entities)"
+   ]
+  }
+ ],
```

Comment:
If you look at our existing tutorial notebooks, they all have a cell at the very end that serves as a "unit test". This final cell checks that the output of the code is actually good, eg. by comparing the indices cleanlab has identified as label issues against indices of known label issues in the dataset.  I recommend we do something similar here, but if you like, it can wait until this notebook is more finalized.